index.js

```diff
@@ -71,6 +84,13 @@ const meow = (helpText, options) => {
 		};
 	}
 
+	if (minimistOptions.array !== undefined) {
+		minimistOptions.array = arrify(minimistOptions.array).map(flagKey => ({
```

Comment:
Can you add a short code comments what's happening here?

Reply:
Sure

Reply:
Better yet: Poke @vadimdemedes to give his input on supporting this in `minimist-options`: [`minimist-options` | Support boolean and number arrays](https://github.com/vadimdemedes/minimist-options/issues/18). Then there wouldn't be any need for a comment (or code!) at all...

Your input would be highly valuable as well!